dbms/src/Encryption/RateLimiter.h

```diff
@@ -412,13 +412,14 @@ class IOLimitTuner
     };
     Watermark writeWatermark() const
     {
-        return getWatermark(writePct());
+        return getWatermark(fg_write_stat, bg_write_stat, writePct());
     }
     Watermark readWatermark() const
     {
-        return getWatermark(readPct());
+        return getWatermark(fg_read_stat, bg_read_stat, readPct());
     }
```

Comment:
we need to tune bg and fg, why revise those two functions? Should we revise https://github.com/pingcap/tiflash/blob/187a46c19267746d1f0a89234bece30ea1ff7eee/dbms/src/Encryption/RateLimiter.cpp#L819-L820

Reply:
thank you, I got it.